ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-joins.json

```diff
@@ -1227,41 +1238,41 @@
       ],
       "post": {
         "topics": {
-          "topics" : [
+          "topics": [
```

Comment:
nit: can we order these topics in a more readable fashion? Previously it was repartition topics first, then changelogs, ordered from left to right.